exporters/jaeger-thrift/src/main/java/io/opentelemetry/exporter/jaeger/thrift/JaegerThriftSpanExporter.java

```diff
@@ -41,7 +41,8 @@
   private static final String IP_KEY = "ip";
   private static final String IP_DEFAULT = "0.0.0.0";
   private final ThriftSender thriftSender;
-  private final Process process;
+  // Visible for testing
+  final Process process;
```

Comment:
I would expose a package protected getter for testing, to keep the logic that all members should be `private final` when possible.